src/pages/docs/main-concepts/3. me-resource.md

```diff
@@ -0,0 +1,21 @@
+---
+path: "/me-resource"
+title: Me Resource
+date: 2019-06-24T23:33:21.852Z
+---
+
+## Me Resource
+Multi-tiered assignments make up a powerful mechanism for customizing the buyer experience down to the individual user. But once configured, the buyer (and developer for that matter) wants a fast and intuitive experience that's more concerned with what they see rather than how or why they see it.
+
+
+## Easy access to what the currently authenticated user owns
+OrderCloud provides the `Me` resource whose job it is to flatten these assignments for the currently authenticated user and just "show me what I can see" regardless at which level these assignments are made.
+
+For example let's say your buyer user is assigned an address at the buyer level, and another address at the usergroup level. Instead of listing assignments between the user and both those levels and manually resolving the addresses yourself, you can simply make a list call as that buyer user to [me/addresses](TODO:link-to-me-addresses) and quickly get the assigned addresses in one quick and fluid motion.
+
+## Ability to admin personal resources
+In addition to being able to quickly get information about the currently authenticated user, the Me resource provides a set of admin endpoints for personal resources (Credit Cards and Addresses) that only that user can see. Not even admins have access to viewing the data created by these users without something like [impersonation](TODO:link-to-impersonation-feature).
+
+
+## Overview
+We've seen how assignments can be made and flow down to the individual user, and how easy it is by using the Me perspective to get those assignments. In the next section we're going to talk about Security Profiles which let you lock down which endpoints a user can access.
```

Comment:
Should we use the same syntax here for `Me` like you used in the second paragraph?